### PR TITLE
Feat/circle ci troubleshoot

### LINKED ISSUE #14: Resolve CircleCI issues with testing

### PR DESCRIPTION
**Pull request by lkarpins**
** Resolves #14 **

** Describe the functionality to be tested: **
- Config.yml modified to address testing/build issue 

** Giphy of how you feel submitting this PR: **

### CODE REVIEW

File: src/Components/Routes/Routes.js [hunk 1]
```diff
@@ -1,28 +1,27 @@
 import React from "react";
-import { Route, Router, Switch } from "react-router-dom";
+import { Route } from "react-router-dom";
```
Comment (catlynbowles) [inline src/Components/Routes/Routes.js:2]: Note to remove unused variables in the future as it will interfere with continuous integration 🐲

**Review by catlynbowles — APPROVED** [review]

File: .circleci/config.yml [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
           paths:
             - .
+
 workflows:
-  test_my_app:
+  test_roed_to_aid_fe:
     jobs:
-      - build_and_test
+      - cypress/run:
+          start: npm start
```
Comment (Rallen13) [inline .circleci/config.yml:25]: This makes much more sense for Cypress.

File: .circleci/config.yml [hunk 3]
```diff
@@ -1,26 +1,26 @@
...
           command: npm run build
-          name: Build app
       - persist_to_workspace:
           root: ~/project
           paths:
             - .
+
 workflows:
-  test_my_app:
+  test_roed_to_aid_fe:
```
Comment (Rallen13) [inline .circleci/config.yml:22]: So thats where we add our project name!

File: .circleci/config.yml (hunk 2 above)
Comment (irmakerdem) [inline .circleci/config.yml:25]: `npm start` to start the app and then run Cypress 😈

**Review by irmakerdem — APPROVED** [review]

File: src/Components/Routes/Routes.js (hunk 1 above)
Comment (Rallen13) [reply src/Components/Routes/Routes.js:2]: I second Catlyns comment. Good to know.

File: src/Components/Routes/Routes.js [hunk 4]
```diff
@@ -1,28 +1,27 @@
...
-        </Route>
-        <Route exact path="/states-policies">
-          <StatePoliciesView />
-        </Route>
-        <Route exact path="/">
-          <HomeView />
-        </Route>
-      </Switch>
-    </Router>
+    <Route>
```
Comment (Rallen13) [inline src/Components/Routes/Routes.js:11]: Route instead of routes! We were close with our troubleshooting.

**Review by Rallen13 — COMMENTED** [review]
Looks great! Great work yall on figuring this out! Glad it wasn't too crazy and no upgrade in versioning of Router required.

**Pull request merged**